plugins/services/src/js/data/groups/resolvers.ts

```diff
@@ -23,12 +29,43 @@ const isGroupArgs = (
   return (args as ServiceGroupQueryArgs).id !== undefined;
 };
 
+function getQuotaLimit(
+  roles: ServiceGroupQuotaRoles | undefined
+): ServiceGroupQuotaLimit {
+  if (roles === undefined) {
```

Comment:
@TattdCodeMonkey , shouldn't be also check for null?

Reply:
yes we should